sys/kern/Cargo.toml

```diff
@@ -33,6 +33,7 @@ phash-gen = {path = "../../build/phash-gen"}
 
```

Comment:
nit: Comment on commit message. This project doesn't seem to use "Signed-off-by" lines in commit message. I also had accidentally added my S-o-b to one of the commits because of old habits. I don't really have a strong preference either ways, but I think we should maintain consistency with what the project does.